src/ValidationException.php

```diff
@@ -2,4 +2,22 @@
 
 namespace Fliglio\Web;
 
-class ValidationException extends \Exception {}
\ No newline at end of file
+use Fliglio\Http\Exceptions\BadRequestException;
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+use Throwable;
+
+class ValidationException extends BadRequestException {
+	/** @var ConstraintViolationListInterface */
+	private $constraintViolationList;
+	public function __construct(ConstraintViolationListInterface $constraintViolationList) {
```

Comment:
this is not backwards compatible, however it was only being used in the validation trait & thus always only accepting a `ConstraintViolationListInterface`